### DISCUSSION TITLE
How to set the default colour theme for the Code plugin?

### DISCUSSION THREAD

**Discussion by purpleyay** [yoopta-editor/Yoopta-Editor#245] [Q&A]
Is there a way to set all Code blocks to default to a specific code theme?  

From looking at [this line](https://github.com/Darginec05/Yoopta-Editor/blob/ff17447cb4d618d0596be80f4f1f4c47889ad939/packages/plugins/code/src/ui/Code.tsx#L35), it looks like it either chooses the specific block's theme or defaults to `VSCode`, but wanted to see if I missed something!

**Comment by Darginec05** [comment]
Hi @purpleyay 
Do you mean to change the default `theme` for the entire plugin?

**Comment by Darginec05** [comment]
@purpleyay check it in examples [here](https://yoopta.dev/examples/withCustomElementProps) ˆ-ˆ
